package.json

```diff
@@ -12,7 +12,7 @@
     "packages/*"
   ],
   "devDependencies": {
-    "@appsignal/types": "^2.1.3",
+    "@appsignal/types": "^2.1.4",
```

Comment:
Looks like you need to update the types package in some files? And the lock file.

Reply:
True! Fixed already.